src/lib/css/plex-card.scss

```
@@ -0,0 +1,211 @@
+@import './variables.scss';
+
+plex-card {
+
+    @mixin cardBasic {
+        cursor: pointer;
+        box-sizing: border-box;
+    }
+    
+    // Estados
+    > div:hover {
```

Haría referencia mediante una clase y no una tag. Como paso con las tabs, si querés cambiar a futuro de div a section (u otro elemento) no necesitas cambiar CSS.

```suggestion
    > .card:hover {
```